drupal/web/modules/custom/wunder_next/src/Commands/WunderNextCommands.php

```diff
@@ -78,7 +80,7 @@ public function setupUserAndConsumer(array $options = ['secret' => '']) {
       'description' => 'This consumer was created by the wunder_next:create-user-and-consumer drush command.',
       'is_default' => FALSE,
       'user_id' => $account->id(),
-      'roles' => [self::API_USER_ROLE],
+      'roles' => [self::API_USER_ROLE, self::FRONTEND_LOGIN_ROLE],
```

Comment:
The consumer we are adding will accept also users that have the frontend login role. TO CHECK: I think this is a security issue, because the users will have the API role, which has administer content etc.